Week-6/es6module/index.js

```diff
@@ -0,0 +1,7 @@
+import {add, subtract} from "./calculator.js";
+
+const sum = add(2, 3);
```

Comment:
As a best practice, add some error handling for edge cases, such as non-numeric inputs.